spring-cloud-dataflow-apps-generator-plugin/src/main/java/org/springframework/cloud/dataflow/app/plugin/generator/AppDefinition.java

```diff
@@ -27,6 +27,12 @@
  */
 public class AppDefinition {
 
+	/**
+	 * Marker 'configClass' value that indicates that this is an autoconfiguration and the @Import statement should be
```

Comment:
Javadoc typically does not like `@` - good to use `{@code @Import}` instead. We must be ignoring Javadoc failures.